lua/lazyvim/plugins/extras/coding/avante.lua

```diff
@@ -0,0 +1,32 @@
+return {
+  {
+    "yetone/avante.nvim",
+    event = "VeryLazy",
+    dependencies = {
+      "stevearc/dressing.nvim",
+    },
+    opts = {
+      hints = { enabled = false },
+    },
+    build = LazyVim.is_win() and "powershell -ExecutionPolicy Bypass -File Build.ps1 -BuildFromSource false" or "make",
+  },
+  {
+    "MeanderingProgrammer/render-markdown.nvim",
+    optional = true,
+    ft = function(_, ft)
+      vim.list_extend(ft, { "Avante" })
+    end,
+    opts = function(_, opts)
+      opts.file_types = vim.list_extend(opts.file_types or {}, { "Avante" })
+    end,
```

Comment:
No need for this since v7.8.0 https://github.com/MeanderingProgrammer/render-markdown.nvim/commit/cb9a5e2412d21c7a89627e0d6da5459acbc0eb9c

Reply:
@dangra 
I've tried to remove this line, but it didn't work since that commit doesn't support the case when `ft` is a function:
https://github.com/MeanderingProgrammer/render-markdown.nvim/commit/cb9a5e2412d21c7a89627e0d6da5459acbc0eb9c#diff-0a534dfa79afd0fc6225ed86d9dec59f8d76f56e559792a9ad3c9bd0f853ba04R22-R29